node/common/core/lib/success.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+function MessageAccepted (transportObj) {
```

Comment:
this is a first stab at creating transport-agnostic result objects for all actions (open/close connection, send events, etc). it's a little empty... except for the transport-specific stuff. but it does have the benefit of allowing to write transport-agnostic code if desired. doesn't feel very elegant though